keras_hub/src/models/backbone.py

```diff
@@ -91,21 +91,24 @@ def get_config(self):
         }
 
         # Add quantization support by utilizing `DTypePolicyMap`
+        dtype = None
```

Comment:
Can't we do dtype = self.dtype_policy as the base case here? if we are trying to write out dtype policy? And leave the code below for quantized cases.

It would be great for explicitness and browsability on kaggle/hf if most of our models just said dtype = bfloat16 (or whatever we actually saved at)